### PR TITLE
dApp: RDN token withdrawal

### LINKED ISSUE #2: Basic development environment set-up
##  Description
The basic development environment is set up.

## Acceptance criteria
- Github is set up
- Basic CI to run the checks are working
- A POC test is implemented
- A basic readme is available
- Basic requirements are included
- The build is split up
- Prettier, ESlint, TypeScript, Web3, Webpack, Jest, Jasmin

## Tasks
- [x] Set up repository
- [x] Check with @karlb about mono repo built
- [x] Add license
- [x] Create readme
- [x] Requirements SDK
- [x] Vue set up
- [x] Copy CircleCI from WebUI and adjust
- [x] Define requirements for Wallet
- [x] Add dev. requirements

## Story Points
5

### PR DESCRIPTION
**Pull request by taleldayekh**
Fixes #2

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-dapp/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 ### Fixed
-- [#2026] Removed delay time on tooltips
+- [#2237] Removed delay time on tooltips
```
Comment (andrevmatos) [inline raiden-dapp/CHANGELOG.md:5]: Wut?

Comment (taleldayekh) [reply raiden-dapp/CHANGELOG.md:5]: Yes, something went wrong. I'm fixing it. :)

File: raiden-dapp/src/components/account/Withdrawal.vue [hunk 2]
```diff
@@ -135,18 +138,28 @@ import Spinner from '@/components/icons/Spinner.vue';
...
+  udcToken!: Token;
   balances: Token[] = [];
   loading: boolean = true;
   withdrawing: boolean = false;
   withdraw: Token | null = null;
 
   async mounted() {
     this.balances = await this.$raiden.getRaidenAccountBalances();
+
+    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
```
Comment (andrevmatos) [inline raiden-dapp/src/components/account/Withdrawal.vue:156]: you're not testing the raiden account balance for udcToken, you're testing just udcToken, which probably is the main account balance

File: raiden-dapp/src/components/account/Withdrawal.vue [hunk 3]
```diff
@@ -135,18 +138,28 @@ import Spinner from '@/components/icons/Spinner.vue';
...
+      this.balances = uniqBy(
+        [...this.balances].concat(this.udcToken),
```
Comment (andrevmatos) [inline raiden-dapp/src/components/account/Withdrawal.vue:158]: I think the proper place to fetch the udcToken balance for the raiden account is in `getRaidenAccountBalances()`, just ensure the address is added there and balance is fetched together with the others.

File: raiden-dapp/src/components/account/Withdrawal.vue (hunk 2 above)
Comment (taleldayekh) [reply raiden-dapp/src/components/account/Withdrawal.vue:156]: Yes, one sec, I'm making a new PR. Stuff accidentally committed the wrong stuff. :)

**Comment by taleldayekh** [comment]
Closing this in favor of a new PR, I messed some things up here...

**Pull request closed without merging**